### PR TITLE
refactor: skip withGraphql options

### LINKED ISSUE #34: Improve Graphql
It would be great if we can make graphql context to be auto detected.

### PR DESCRIPTION
**Pull request by 9renpoto**
closes #34 

refs: https://github.com/nestjs/graphql/issues/705#issuecomment-604579768

### CODE REVIEW

File: lib/raven.interceptor.ts [hunk 1]
```diff
@@ -15,23 +15,23 @@ import {
...
+  constructor(private readonly options: IRavenInterceptorOptions = {}) {}
 
   intercept(context: ExecutionContext, next: CallHandler): Observable<any> {
     // first param would be for events, second is for errors
     return next.handle().pipe(
       tap(null, (exception) => {
         if (this.shouldReport(exception)) {
           Sentry.withScope((scope) => {
-            switch (context.getType()) {
+            switch (context.getType<GqlContextType>()) {
```
Comment (9renpoto) [inline lib/raven.interceptor.ts:34]: refs: https://github.com/nestjs/graphql/issues/705#issuecomment-604579768

File: lib/raven.interfaces.ts [hunk 2]
```diff
@@ -15,7 +15,6 @@ export interface IRavenInterceptorOptions {
   extra?: { [key: string]: any };
   fingerprint?: string[];
   level?: Severity;
-  withGraphQL?: boolean;
```
Comment (9renpoto) [inline lib/raven.interfaces.ts:18]: remove option

**Comment by 9renpoto** [comment]
@mentos1386 Hi, remove `withGraphQL` options. please review it :). thanks!

File: example/src/app.module.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ import { GqlModule } from './gql/gql.module';
 @Module({
   imports: [
     GraphQLModule.forRoot({
-      autoSchemaFile: 'example/src/schema.gql',
+      autoSchemaFile: true,
```
Comment (9renpoto) [inline example/src/app.module.ts:12]: skip generate `schema.gql` ([refs](https://docs.nestjs.com/graphql/quick-start#code-first))

**Comment by 9renpoto** [comment]
@mentos1386 Hi, please review it. thank you :)

**Comment by mentos1386** [comment]
Thanks! I don't like that we need graphql as dependency. But i don't think there is a better way atm.

**Comment by 9renpoto** [comment]
Thank you merging !

**Pull request merged**